src/anime/anime.ts

```diff
@@ -0,0 +1,49 @@
+export interface Anime {
+  mal_id: number;
+  title: string;
+  images: {
+    jpg: {
+      image_url: string;
```

Comment:
I see pther part where you jse ?. But there will not be an exception if field does not exist?